catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/cql/CqlRequestTest.java

```diff
@@ -152,6 +154,28 @@ public void testBlankLocalSource() {
     assertThat(cqlRequest.getSource(), is("source"));
   }
 
+  @Test
+  public void testMultipleSources() {
+    String[] sources = {"source1", "source2"};
+    cqlRequest.setSrcs(sources);
+    cqlRequest.createQueryRequest("source1", filterBuilder);
+    assertThat(Arrays.asList(cqlRequest.getSrcs()), contains("source1", "source2"));
+  }
+
```

Comment:
Might be worth considering adding a test case where "source1" is all-caps like `SOURCE1`